### PR TITLE
TextView extensions

### PR DESCRIPTION
**Pull request by dovahkiin98**

### CODE REVIEW

**Comment by dovahkiin98** [comment]
Signed in

**Comment by JakeWharton** [comment]
Shouldn't these all be on `TextView`? `EditText` is an empty class.

**Comment by dovahkiin98** [comment]
@JakeWharton I thought it would be more useful to have these methods on EditText since the getText() returns editable, and TextWatcher is used more on EditText

**Comment by davethomas11** [comment]
Duplicates #151 #229

File: src/main/java/androidx/view/EditText.kt [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+ * Set and Get the string value of EditText content
+ */
+var EditText.content
+    get() = text.toString()
+    set(value) {
+        setText(value)
+    }
+
+/**
+ * @returns Weather the EditText text is empty or not
```
Comment (pavs23) [inline src/main/java/androidx/view/EditText.kt]: Do you mean "Weather"?

Comment (davethomas11) [reply src/main/java/androidx/view/EditText.kt]: Rain or shine, the EditText is empty or not.

Comment (pavs23) [reply src/main/java/androidx/view/EditText.kt]: Haha 😂

File: src/main/java/androidx/view/TextView.kt [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+fun TextView.clear() {
+    text = ""
+}
+
+/**
+ * @param before action performed before text changes
+ * @param after action performed after text changes
+ * @param onTextChanged action preformed on text change
+ */
+fun TextView.onTextChanged(
```
Comment (JakeWharton) [inline src/main/java/androidx/view/TextView.kt]: This should be `inline` and the lambdas should be `crossinline`.

File: src/main/java/androidx/view/TextView.kt [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+/**
+ * @returns Whether the EditText text is blank or not
+ */
+val TextView.isBlank get() = text.isBlank()
+
+/**
+ * Clears the contents of the EditText
+ */
+fun TextView.clear() {
```
Comment (JakeWharton) [inline src/main/java/androidx/view/TextView.kt]: I'm not sure these 4 extensions have any real value. They're conveniences for things which are already pretty convenient.

**Comment by 79joey** [comment]
Can you please have some off your mobile application for my project at college

**Comment by JakeWharton** [comment]
Text watcher-based listeners have been merged in http://r.android.com/720364. As I said above, I don't think the other extensions are worth including since they're so trivial and don't leverage any advantage of the language.

**Pull request closed without merging**